Riot/Modules/Rooms/ShowDirectory/Cells/Room/DirectoryRoomTableViewCellVM.swift

```diff
@@ -0,0 +1,62 @@
+// 
+// Copyright 2020 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+
+struct DirectoryRoomTableViewCellVM {
+    private let room: MXPublicRoom
```

Comment:
Can we remove this dependency and the one to `MXSession`?
This class should be just a placeholder populated externally  like

```

struct DirectoryRoomTableViewCellVM {
    let title: String?
    let numberOfUsers: Int
    let title: String?
...
}